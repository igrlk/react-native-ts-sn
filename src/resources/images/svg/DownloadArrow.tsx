import React from 'react';
import Svg, { Path } from 'react-native-svg';

export default function DownloadArrow() {
	return (
		<Svg width='22' height='22' viewBox='0 0 22 22' fill='none'>
			<Path
				d='M16.6593 11.4979C16.8935 11.2638 16.8935 10.8951 16.6593 10.6609C16.433 10.4267 16.0564 10.4267 15.8307 10.6609L11.5865 14.8972V0.899829C11.5865 0.573352 11.327 0.30542 11.0005 0.30542C10.6741 0.30542 10.4061 0.573352 10.4061 0.899829V14.8972L6.17036 10.6609C5.93618 10.4267 5.55907 10.4267 5.33333 10.6609C5.09916 10.8951 5.09916 11.2638 5.33333 11.4979L10.5823 16.7469C10.8085 16.9731 11.1851 16.9731 11.4193 16.7469L16.6593 11.4979ZM20.8196 15.3408C20.8196 15.0143 21.0791 14.7548 21.4056 14.7548C21.7321 14.7548 22 15.0143 22 15.3408V21.1003C22 21.4267 21.7321 21.6947 21.4056 21.6947H0.58597C0.259494 21.6947 0 21.4267 0 21.1003V15.3408C0 15.0143 0.259494 14.7548 0.58597 14.7548C0.912447 14.7548 1.18038 15.0143 1.18038 15.3408V20.5143H20.8196V15.3408Z'
				fill='#9E9E9E'
			/>
		</Svg>
	);
}
