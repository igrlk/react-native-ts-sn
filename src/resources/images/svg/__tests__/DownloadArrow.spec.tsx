import React from 'react';
import renderer from 'react-test-renderer';

import DownloadArrow from '../DownloadArrow';

describe('DownloadArrow', () => {
	it('match to snapshot', () => {
		const tree = renderer.create(<DownloadArrow />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
