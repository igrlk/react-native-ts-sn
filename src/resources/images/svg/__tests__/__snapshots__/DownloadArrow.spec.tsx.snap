// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DownloadArrow match to snapshot 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="22"
  bbWidth="22"
  fill="none"
  focusable={false}
  height="22"
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      undefined,
      Object {
        "opacity": 1,
      },
      Object {
        "flex": 0,
        "height": 22,
        "width": 22,
      },
    ]
  }
  vbHeight={22}
  vbWidth={22}
  width="22"
>
  <RNSVGGroup
    fill={null}
    fillOpacity={1}
    fillRule={1}
    font={Object {}}
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={
      Array [
        "fill",
      ]
    }
    rotation={0}
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    x={0}
    y={0}
  >
    <RNSVGPath
      d="M16.6593 11.4979C16.8935 11.2638 16.8935 10.8951 16.6593 10.6609C16.433 10.4267 16.0564 10.4267 15.8307 10.6609L11.5865 14.8972V0.899829C11.5865 0.573352 11.327 0.30542 11.0005 0.30542C10.6741 0.30542 10.4061 0.573352 10.4061 0.899829V14.8972L6.17036 10.6609C5.93618 10.4267 5.55907 10.4267 5.33333 10.6609C5.09916 10.8951 5.09916 11.2638 5.33333 11.4979L10.5823 16.7469C10.8085 16.9731 11.1851 16.9731 11.4193 16.7469L16.6593 11.4979ZM20.8196 15.3408C20.8196 15.0143 21.0791 14.7548 21.4056 14.7548C21.7321 14.7548 22 15.0143 22 15.3408V21.1003C22 21.4267 21.7321 21.6947 21.4056 21.6947H0.58597C0.259494 21.6947 0 21.4267 0 21.1003V15.3408C0 15.0143 0.259494 14.7548 0.58597 14.7548C0.912447 14.7548 1.18038 15.0143 1.18038 15.3408V20.5143H20.8196V15.3408Z"
      fill={
        Array [
          0,
          4288585374,
        ]
      }
      fillOpacity={1}
      fillRule={1}
      matrix={
        Array [
          1,
          0,
          0,
          1,
          0,
          0,
        ]
      }
      opacity={1}
      originX={0}
      originY={0}
      propList={
        Array [
          "fill",
        ]
      }
      rotation={0}
      scaleX={1}
      scaleY={1}
      skewX={0}
      skewY={0}
      stroke={null}
      strokeDasharray={null}
      strokeDashoffset={null}
      strokeLinecap={0}
      strokeLinejoin={0}
      strokeMiterlimit={4}
      strokeOpacity={1}
      strokeWidth={1}
      vectorEffect={0}
      x={0}
      y={0}
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
